registration/registration-services/src/main/java/io/mosip/registration/util/acktemplate/TemplateGenerator.java

```diff
@@ -805,20 +810,29 @@ private void setUpDemographicInfo(RegistrationDTO registration, Map<String, Obje
 		Map<String, Object> defaultDemographics = registration.getDefaultDemographics();
 		if (defaultDemographics != null && !defaultDemographics.isEmpty()) {
 			templateValues.put(RegistrationConstants.TEMPLATE_APPLICANT_NAME_PRIMARY_LABEL,
-					applicationLanguageProperties.getString("fullName"));
+					applicationLanguageProperties.getString("applicantname"));
```

Comment:
How are we taking applicant name here @HimajaDhanyamraju2  ?

Reply:
Applicant name is stored in the session based on the group "FullName" in schema.